rafthttp/pipeline.go

```diff
@@ -136,11 +143,11 @@ func (p *pipeline) post(data []byte) (err error) {
 		return err
 	}
 	b, err := ioutil.ReadAll(resp.Body)
+	resp.Body.Close()
```

Comment:
defer doesn't buy anything here; the function won't panic so it'll only be needless overhead.